sdks/python/apache_beam/utils/subprocess_server.py

```diff
@@ -283,6 +290,39 @@ def beam_services(cls, replacements):
     finally:
       cls._BEAM_SERVICES.replacements = old
 
+  @classmethod
+  def make_classpath_jar(cls, main_jar, extra_jars, cache_dir=None):
```

Comment:
Please add a unit test for this as well.

Reply:
Done.